xbmc/settings/SettingConditions.cpp

```diff
@@ -257,6 +258,14 @@ bool ProfileLockMode(const std::string& condition,
   return CSettingConditions::GetCurrentProfile().getLockMode() == lock;
 }
 
+bool IsSplashEnabled(const std::string& condition,
+                     const std::string& value,
+                     const SettingConstPtr& setting,
+                     void* data)
+{
+  return CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_splashImage;
+}
```

Comment:
I prefer to change only what is necessary, maybe in the next one, thanks

Reply:
@joseluismarti has a good point, while it would be welcome, no need (and much safer) to not touch existing code in this PR.

And it's worth mentioning a follow-up PR that moves global symbols into anonymous namespaces would be appreciated!

Reply:
:+1:  No problem to do this follow-up PR